client/src/cordova/apple/OutlineAppleLib/Sources/OutlineTunnel/OutlineVpn.swift

```diff
@@ -52,15 +53,13 @@ public class OutlineVpn: NSObject {
   /** Starts a VPN tunnel as specified in the OutlineTunnel object. */
   public func start(_ tunnelId: String, named name: String?, withTransport transportConfig: String) async throws {
     if let manager = await getTunnelManager(), isActiveSession(manager.connection) {
-      DDLogDebug("Stoppping active session before starting new one")
```

Comment:
Please do not remove these logs. They are very helpful when debugging the plugin.